src/vm/builtins/builtin_runner/range_check.zig

```diff
@@ -78,6 +78,10 @@ pub const RangeCheckBuiltinRunner = struct {
         };
     }
 
+    pub fn initDefault() Self {
```

Comment:
it would be good to have a bit of a huddle/discussion about how we want to initialize builtins w/ the instance type defs 

my proposal/take:

smol refactor to `range_check_instance_def` where
-  we use default field values: https://ziglang.org/documentation/master/#Default-Field-Values
-  remove current init
-  `from` refactored to `init`
- initialize RangeCheckInstanceDef struct and use ratio/n_parts fields in defaultInit

lmk if this is clear/sounds good 🦆

Reply:
Sounds good. But what do you mean by ?

> from refactored to init

Reply:
Also, in terms of general practice, I also like this point 
https://github.com/keep-starknet-strange/ziggy-starkdust/pull/249#discussion_r1429181712

Reply:
> Sounds good. But what do you mean by ?
> 
> > from refactored to init

Great! Here I meant specifically for 

https://github.com/keep-starknet-strange/ziggy-starkdust/blob/c65c9dd984e05aef33909bc99bd9b0628d5043eb/src/vm/types/range_check_instance_def.zig#L34-L39

we would refactor `from` to `init` here

Reply:
Ohh yes, that's great! Sounds good to me